sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -673,55 +722,69 @@ def plot(self, ax=None, n_cols=3, line_kw=None, contour_kw=None):
                 preds = pd_result.individual
 
             if len(values) == 1:
```

Comment:
This `values` local variable could be renamed `feature_values` to make the code easier to follow.